public/_redirects

```diff
@@ -0,0 +1 @@
+/*  /index.html 200
```

Comment:
Netlify uses the netlify.toml for redirects from what I am seeing in our doc site. Does this work?